shimmerlayout/src/main/java/io/supercharge/shimmerlayout/ShimmerLayout.java

```diff
@@ -310,4 +327,63 @@ private void removeGlobalLayoutListener(ViewTreeObserver.OnGlobalLayoutListener
     private int reduceColorAlphaValueToZero(int actualColor) {
         return Color.argb(0, Color.red(actualColor), Color.green(actualColor), Color.blue(actualColor));
     }
+
+    private Rect calculateMaskRect() {
+        if (shimmerAngle == 0) {
+            return new Rect(0, 0, getWidth(), getHeight());
```

Comment:
@szugyi if the angle is zero, then the width can be the original getWidth() / 2 since it will be a vertical line, right?